scripts/README.md

```diff
@@ -119,4 +120,21 @@ Get byte count for a specific EVC:
 22594175814
 ```
 
+Get tables statistics:
```

Comment:
Great to have the examples. Let's not forget to update the CHANGELOG.rst too, summarizing what has been added. 

I'll leave the rest of the review to @italovalcy, since he knows best how this script and its output is used in production.

Reply:
@viniarck, Thank you.